nebari/template/stages/02-infrastructure/aws/main.tf

```diff
@@ -18,6 +26,8 @@ module "accounting" {
 
 # ======================= NETWORK ======================
 module "network" {
+  count = local.override_network ? 0 : 1
```

Comment:
Is this being used anywhere?

Reply:
This is a feature which prevents this module from running if local.override_network is True
https://developer.hashicorp.com/terraform/language/meta-arguments/count